module/VuFind/src/VuFind/ILS/Driver/KohaRest.php

```diff
@@ -182,7 +182,6 @@ class KohaRest extends \VuFind\ILS\Driver\AbstractBase implements
      */
     protected $itemStatusMappings = [
         'Item::Held' => 'On Hold',
-        'Item::Lost' => 'Lost--Library Applied',
```

Comment:
You've deleted this code but not added it anywhere else. Should this be used as my proposed example in the .ini file? Do we need a mechanism for configuring a global fallback string independent of the numeric codes?

Reply:
It's now right below in itemStatusMappingMethods.

Reply:
Sorry for being unclear; I was referring specifically to the text "Lost--Library Applied" rather than the 'Item::Lost' code itself.

Reply:
Right. Now the code takes the default text if there's no mapping so I thought it would make sense to use that. I can add that term as an example, that's a good idea.